kiwi/cli/pipelines/train.py

```diff
@@ -113,7 +113,8 @@ def train_opts(parser):
         const=True,
         default=False,
         help='Resume training a previous run. '
-        'The --run-uuid (and possibly --experiment-name) '
+        'Kiwi looks for the trainer in the --output-dir. '
```

Comment:
Let me see if I understand his correctly (it's a good test as I forgot what the logic was ):
(i) if output-dir is given, look for trainer there (with what name?)
(ii) if output-dir is not given, run-uuid must be specified and kiwi looks in runs/run-uuid

Reply:
I suggest to write it like that:

'(i) If --output-dir is not None, kiwi tries to load output-dir/MODEL_CONSTANT_NAME'
(ii) If 'ouptut-dir' is None, run-uuid has to be set. (iii) Kiwi tries to retrieve the run by ID from MLFlow: If `runs` is a local directory, kiwi looks in runs/run-uuid/MODEL_CONSTANT_NAME,
If `runs` is not  a local directory, kiwi retrieves runs/run-uuid folder from mlflow.